frontend/src/components/graduations/GraduationCard.tsx

```diff
@@ -0,0 +1,278 @@
+                    <ArrowDownwardIcon
+                        sx={{
+                            fontSize: '2.25rem',
+                            fontWeight: 'bold',
+                            mt: '-3px',
+                        }}
+                        color='error'
+                    />
+                )}
+
+                <Typography
+                    alignContent={center ? 'center' : 'left'}
+                    sx={{
+                        fontSize: '2.25rem',
+                        lineHeight: 1,
+                        fontWeight: 'bold',
+                    }}
+                    color={value >= 0 ? 'success.main' : 'error.main'}
+                >
+                    {Math.abs(value)}
+                </Typography>
+            </Stack>
+        </Stack>
+    );
+}
+
+interface GraduationCardProps {
+    graduation: Graduation;
+}
+
+export default function GraduationCard({ graduation }: GraduationCardProps) {
+    const {
+        newCohort,
+        ratingSystem,
+        score,
+        progress,
+        currentRating,
+        startRating,
+        displayName,
+    } = graduation;
+
+    const hours =
+        Object.values(progress)
+            .flatMap((reqProg) => Object.values(reqProg.minutesSpent))
+            .reduce((a, b) => a + b, 0) / 60;
+
+    const historyData = useMemo(() => getChartData(graduation), [graduation]);
+
+    const finalRating = currentRating;
+    const ratingChange = finalRating - startRating;
+
+    return (
+        <Box
+            width='800px'
+            height='540px'
+            display='grid'
+            gap='0.5rem'
+            paddingY='32px'
+            paddingX='64px'
+            gridTemplateColumns='1fr auto'
+            gridTemplateRows='auto max-content auto'
+            gridTemplateAreas={['"header blank"', '"chart dojo"', '"stats empty"'].join(
+                '\n',
+            )}
+        >
+            <Stack
+                direction='row'
+                flexWrap='wrap'
+                justifyContent='center'
+                alignItems='center'
+                columnGap='2ch'
+                gridArea='header'
+            >
+                <Box>
+                    <Stack direction='row' columnGap='1ch' flexWrap='wrap'>
+                        <Typography lineHeight={1} variant='h5'>
+                            Congrats{' '}
+                        </Typography>
+                        <Typography lineHeight={1} variant='h5' color='dojoOrange.main'>
+                            {' '}
+                            {displayName}
+                        </Typography>
+                    </Stack>
+                    <Stack direction='row' columnGap='1ch' flexWrap='wrap'>
+                        <Typography lineHeight={1} variant='h5'>
+                            on graduating to
+                        </Typography>
+                        <Typography lineHeight={1} variant='h5'>
+                            {newCohort}!
```

Comment:
Why is this in a separate typography? This can be in the same one as `on graduating to` right?

Reply:
This segmentation is to control where text chunks wrap.